### DISCUSSION TITLE
How  to distinguish in which panel user have opened page ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#13932] [Help]
### Package

Panel builder

### Package Version

^3.2-stable

### How can we help you?

In laravel 10 / filamentphp 3 app Having Admin/Managers panels if there is a way to distinguish in which panel user have opened page with : 
a) Different title on Dashboard page
b) Different title on any page header, like : 'Managers panel category listing'
c) Different layout colors, fonts to see at first glance which area is opened ?

Something easy to install / use...

**Comment by leandrocfe** [comment]
you can get the current panel using `filament()->getCurrentPanel()`

**Reply by sergeynilov** [reply]
But how to apply  changes for a panel ?

**Reply by leandrocfe** [reply]
You can customize the page titles by using conditions to display different titles based on the panel.
You can apply unique layout colors and fonts to each panel https://filamentphp.com/docs/3.x/panels/themes#changing-the-colors

**Reply by sergeynilov** [reply]
Thanks! I found how to change layout colors! How to change title on Dashboard page/ Different title on any page header ?
